src/main/java/li/cil/tis3d/common/LocalAPI.java

```diff
@@ -0,0 +1,13 @@
+package li.cil.tis3d.common;
+
+import li.cil.tis3d.api.CommonAPI;
+import li.cil.tis3d.api.ClientAPI;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+public class LocalAPI {
+    public static CommonAPI common;
+
+    @Environment(EnvType.CLIENT)
+    public static ClientAPI client;
```

Comment:
This entire PR is a bit lazy, but this was lazier than normal. I think instead of just reusing the bundle classes here, I should have put the API classes directly embedded here, which would save us an additional indirection.

Reply:
Well, this way it only needs a single `@Environment`, and makes it clear from the name alone that it's client only stuff, so I actually like it :)